libopenage/engine/engine.h

```diff
@@ -72,11 +74,13 @@ class Engine {
 	 * @param root_dir openage root directory.
 	 * @param mods The mods to load.
 	 * @param debug_graphics If true, enable OpenGL debug logging.
+	 * @param window_settings window display setting
```

Comment:
You should describe what the parameter is used for. What you wrote is already obvious from the name.

Also, you should keep the comment style consistent (first letter capitalized, dot at the end).